scss/elements/_lists.scss

```diff
@@ -65,6 +65,10 @@
 			}
 		}
 	}
+
+	&--no-pipe:after {
```

Comment:
Could we call this `no-separator` instead so its not coupled to the `pipe-separated`?

Reply:
That's a good idea

Reply:
Have updated